documentation/modules/ROOT/partials/modules/tutorial/proc-registering-connector-monitor-inventory-database.adoc

```diff
@@ -69,6 +70,14 @@ Upon restart, the connector will recover the schemas of the database that existe
 For more information, see {link-prefix}:{link-mysql-connector}#mysql-connector-properties[MySQL connector configuration properties].
 --
 
+ifdef::community[]
+[NOTE]
+====
+For security reasons, you shouldn't put passwords or other secrets in plain text into connector configurations.
```

Comment:
I'd be more prescriptive and say _"do not"_.

Reply:
Yeah, that's an interesting one. So I think I will keep that, as this isn't strictly needed, you can achieve means of securely running this also otherwise, e.g. by authenticating the REST end point of Kafka Connect and providing access only to a selected set of individuals. So this is not a "must" in the strictest sense.